app.py

```diff
@@ -0,0 +1,128 @@
+from flask import Flask, request, abort
+from git import Repo
+import ansible_runner
+import os
+import requests
+import json
+
+app = Flask(__name__)
+
+def updateRepo(): #Update the local git repo
+    if os.path.isdir('ansible-switches'): #If the clone of the repo exists pull updates, if not clone it 
+        repo = Repo('ansible-switches')
+        o = repo.remotes.origin
+        o.pull()
+        print('pulled')
+    else:
+        Repo.clone_from("https://github.com/CCI-MOC/ansible-switches.git", "ansible-switches")
+        print('cloned')
+
+def fileChanges(commits): #Returns bool for if vlan file changes and list of individual switch changes
+    
+    vlanChange = False #Vlan Flag
+    switchChanges = [] #List of modified switches
+
+    for commit in commits: #Look at each commit in push to main branch
+        
+        for file in commit['modified']: #Look at the files modified in each commit
+            
+            if file == 'group_vars/all/vlans.yaml' : #Set flag to true if vlan change
+                vlanChange = True
+                
+            elif file[:9] == 'host_vars' : #Add all changed switches to a list 
```

Comment:
```suggestion
            elif file.split("/")[0] == 'host_vars' : #Add all changed switches to a list 
```

I would split the string and capture the first element, to ensure that you are comparing only the first parent. I see you split file later, so it might make sense to store the split list higher up